src/html/generator.ml

```diff
@@ -1179,11 +1210,11 @@ module Class :
 sig
   val class_ :
     ?theme_uri:Tree.uri -> Lang.Signature.recursive -> Lang.Class.t ->
-      rendered_item * Odoc_model.Comment.docs * Tree.t list
+      rendered_item * Odoc_model.Comment.docs * toc * Tree.t list
```

Comment:
Is there additional work to do here, as you've added the `toc` to the return type but they're hardcoded to `[]` below?

Reply:
I don't believe so. I think right now it only makes sense for includes to add entries to the ToC.
Classes, modules, etc. are currently only displayed on separate pages.

Reply:
OK, so it's just for uniformity of type?

Reply:
:man_shrugging: 

I followed the current structure of the html generator, and went for minimal updates.
I considered adding a new kind of items : "inlined items", which would have been the only one returning a ToC.
But I think the result might actually have been slightly worse.

Reply:
:+1: